packages/event-listener/src/index.ts

```diff
@@ -1,17 +1,37 @@
 import Web3 from 'web3';
+import axios, { AxiosResponse } from 'axios';
 
-import { EventServiceProvider } from './services/event.service';
+import { EmailServiceProvider, IEmailServiceProvider } from './services/email.service';
+import { IEmailAdapter } from './email/IEmailAdapter';
+import { MandrillEmailAdapter } from './email/mandrill.adapter';
+import { IOriginEventListener, OriginEventListener } from './listeners/origin.listener';
+import { OriginEventsStore, IOriginEventsStore } from './stores/OriginEventsStore';
 
 export const SCAN_INTERVAL = 3000;
 
-const startEventListener = () => {
+const startEventListener = async () => {
     const web3 = new Web3(process.env.WEB3 || 'http://localhost:8550');
+    const backendUrl: string = process.env.BACKEND_URL || 'http://localhost:3035';
 
-    const eventService = new EventServiceProvider(process.env.BACKEND_URL, web3);
+    const result: AxiosResponse = await axios.get(`${backendUrl}/MarketContractLookup`);
```

Comment:
When declaring the const you probably want to skip adding type explicitly. It's obviously correct as it it, perhaps we should agree on the style or/and enforce it via eslint.